.github/workflows/action-conda-publish.yml

```diff
@@ -0,0 +1,78 @@
+name: Anaconda Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  publish:
+    name: publish / python-3.8 / ubuntu-latest
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: goanpeca/setup-miniconda@v1
+        with:
+          activate-environment: esmvaltool
+          auto-update-conda: true
+          environment-file: environment.yml
```

Comment:
The dependencies from environment.yml are not needed to build the package, so I would remove this